packages/web3-eth/src/web3_eth.ts

```diff
@@ -95,52 +95,169 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 			registeredSubscriptions,
 		});
 	}
+
+    /**
+     * @returns Returns the ethereum protocol version of the node.
+     *
+     * ```ts
+     * web3.eth.getProtocolVersion().then(console.log);
```

Comment:
I think it is better to use `await` and not use `.then(...)` to make the code snippet more elegant, here and in all other places. Or what do you think?
```suggestion
     * const version = await web3.eth.getProtocolVersion();
     * console.log(version);
```

Reply:
I thought so as well, but `.then` was used elsewhere by other devs, so was just trying to stay consistent